components/workflow.rst

```diff
@@ -94,6 +94,19 @@ you can retrieve a workflow from it and use it as follows::
     $workflow->can($blogPost, 'publish'); // True
     $workflow->getEnabledTransitions($blogPost); // $blogPost can perform transition "publish" or "reject"
 
+Initialization
+--------------
+
+If you want to initiate your workflow, you can call ``getMarking``::
```

Comment:
```suggestion
If you want to initiate your workflow, you can call ``getMarking()`` method::
```

Reply:
And I'm not super confortable with this sentence. What does mean "initiate your workflow"?
I do understand what you want to say, but for new comers, it could be confusing.

What about something like

> When the object enter the workflow for the first time, the workflow will initialize the `property` with the `initial_place`.
If you want to have always a valid marking in your database, you can use the `getMarking()` method to initialize the object property with the initial place::

Reply:
you are right about initial marking :) I used the old name, sorry.

You sentence is correct, let's go for it.

Minor detail: 
```diff
- in the configuration
+ from the configuration
```

Reply:
DONE ! 

If it's ok for you, I let you resolve the conversation and maybe... merge the PR ?